README.rst

```diff
@@ -21,31 +47,14 @@ type must be one of [build, ci, docs, feat, fix, perf, refactor, revert, style,
 
 Versioning and Releases
 -----
-This library has its version automatically updated by semantic-versioning when the release is published to npm. The source code package.json file should remain at "1.0.0-semantically-released".  The below Usage instructions will install the latest version from npm. Please see the repository Releases for earlier package versions.
+This library has its version automatically updated by Lerna using semantic-versioning under-the-hood when the release is published to npm. The source code package.json file versions should 0.0.0.
 
-Preview next release version from Pull Requests
+Preview changed packages in CI with Github Actions
 *****
 
-As a convenience, the "node_js CI / build (push)" check on Pull Requests includes a step to analyze the commit(s) and outputs a preview of what version semantic-release will publish if a PR gets merged. This is done using the "--dry-run" option for the semantic-release CLI, which will skip the publish/release steps. Look for a message in this CI step along the lines of "The next release version is <NEXT_RELEASE_VERSION>".
-
-Usage
------
-
-To install frontend-enterprise into your project:
-
-::
+As a convenience, the ``lerna changed`` command is run for each push to determine which packages in the monorepo will be published should a PR get merged.
 
-   npm i --save @edx/frontend-enterprise
-
-.. |Build Status| image:: https://api.travis-ci.com/edx/frontend-enterprise.svg?branch=master
-   :target: https://travis-ci.com/edx/frontend-enterprise
-.. |Coveralls| image:: https://img.shields.io/coveralls/edx/frontend-enterprise.svg?branch=master
-   :target: https://coveralls.io/github/edx/frontend-enterprise
-.. |npm_version| image:: https://img.shields.io/npm/v/@edx/frontend-enterprise.svg
-   :target: @edx/frontend-enterprise
-.. |npm_downloads| image:: https://img.shields.io/npm/dt/@edx/frontend-enterprise.svg
-   :target: @edx/frontend-enterprise
-.. |license| image:: https://img.shields.io/npm/l/@edx/frontend-enterprise.svg
```

Comment:
We should probably still be linking to the license in the readme.  Though I'm not sure this ever actually did.

Reply:
Yeah, I decided to remove these from the root README.rst in favor of adding them back into the individual package's README files once the new packages are published on NPM.

This will be a quick follow-up PR once things are published.